app/views/two_factor_authentication/shared/max_login_attempts_reached.html.erb

```diff
@@ -0,0 +1,18 @@
+<% lockout_time_in_words = decorated_user.lockout_time_remaining_in_words %>
```

Comment:
I used a variable here because the view originally had the `decorated_user.lockout_time_remaining_in_words` sitting directly in the markup taking up a bunch of characters 😄. I saw variables in other views, and since I'm still learning how you all write rails code, I just went with that. I certainly don't mind making a helper method in this case if that's the style we're should to be adhering to!